cv/qc-cv.obo

```diff
@@ -18,1967 +18,341 @@ import: https://raw.githubusercontent.com/ISA-tools/stato/dev/releases/1.4/stato
-is_a: MS:4000107 ! Table column type
-relationship: has_units PATO:0000161 ! rate
-
-[Term]
-id: MS:4000115
-name: Count column
-def: "The column contains counts of occurrences of a common type." [PSI:QC]
-is_a: MS:4000107 ! Table column type
-relationship: has_units UO:0000189 ! count unit
-
-[Term]
-id: MS:4000116
-name: Peptide sequence column
-def: "The column contains peptide sequences; multiple peptides are separated by space; sequences in ProForma encoding." [PSI:QC]
-comment: ProForma v2 - https://github.com/HUPO-PSI/ProForma/
-is_a: MS:4000107 ! Table column type
-relationship: has_type NCIT:C45253 ! String
-
-[Term]
-id: MS:4000117
-name: Outliers below column
-def: "The outlier values below 1.5 * IQR." [PSI:QC]
-is_a: MS:4000107 ! Table column type
-relationship: has_type NCIT:C79083 ! Outlier
-
-[Term]
-id: MS:4000118
-name: Outliers above column
-def: "The outlier values above 1.5 * IQR." [PSI:QC]
-is_a: MS:4000107 ! Table column type
-relationship: has_type NCIT:C79083 ! Outlier
-
-[Term]
-id: MS:4000119
-name: Protein coverage
-def: "The coverage of protein sequences from the peptide sequences identified. The table records the coverage itself (in percent), the protein accession (SHOULD correspond to the accession used in fasta document used for Sequence DB based identification), Length of the protein(, and optionally if it is 'target' or 'decoy' entry.)" [PSI:QC]
-comment: The protein coverage can provide insight into the sensitivity of the instrument/identification/protocols used.
-is_a: MS:4000001 ! QC metric
-is_a: MS:4000009 ! ID based
-is_a: MS:4000006 ! table
-relationship: has_column MS:4000110 ! Sequence coverage column
-relationship: has_column MS:4000111 ! Protein accession column
-relationship: has_column MS:4000112 ! Sequence length column
-relationship: has_optional_column MS:4000113 ! Target or decoy designation column
-
-[Term]
-id: MS:4000120
-name: Sampling rates
-def: "The sampling rates of identified peptides and their respective frequency (accumulated by the sampling rate, i.e. how often peptides were sampled n times and how many peptides were resampled at that rate. Optionally, which peptides were sampled at that rate, space separated)" [PSI:QC]
-comment: The sampling rate may give insights to peptide separation, dynamic exclusion settings and gradient efficiency.
-is_a: MS:4000001 ! QC metric
-is_a: MS:4000009 ! ID based
-is_a: MS:4000006 ! table
-relationship: has_column MS:4000114 ! Sample rate column
-relationship: has_column MS:4000115 ! Count column
-relationship: has_optional_column MS:4000116 ! Peptide sequence column
-
-[Term]
-id: MS:4000121
-name: Explained precursor intensity first quarter
-def: "Fraction of identified MS2 in the first quarter of MS2 sorted by precursor intensity." [PSI:QC]
+name: explained precursor intensity
+def: "Fraction of identified MS2 in the first, second, thrid, and forth quarter of MS2 sorted by precursor intensity at given FDR threshold." [PSI:QC]
```

Comment:
is that the precursor intensity as annotated by the instrument (e.g. the preceding MS1 scan), or an abundance computed by a software for the whole elution&isotope profile of an ion species?